evap/staff/views.py

```diff
@@ -570,8 +570,7 @@ def semester_questionnaire_assign(request, semester_id):
 def semester_preparation_reminder(request, semester_id):
     semester = get_object_or_404(Semester, id=semester_id)
 
-    evaluations = semester.evaluations.filter(state__in=['prepared', 'editor_approved']).prefetch_related("course__degrees")
-
+    evaluations = semester.evaluations.filter(state__in=['prepared', 'editor_approved']).prefetch_related("course__degrees").order_by("full_name_de")
```

Comment:
Seems like a very long line. Consider splitting the statement in multiple lines :)